app.js

```diff
@@ -34,6 +34,19 @@ app.get('/api/v1/palettes/:id', async (request, response) => {
   } catch (error) {
     response.status(500).json({ error })
   }
-})
+});
+
+app.post('/api/v1/projects', async (request, response) => {
+  const project = request.body;
+  if (!project.name) {
+    return response.status(422).json({error: 'Expected body format {name: <String>}. You\'re missing the required name property'});
+  }
+  try {
+    const id = await database('projects').insert(project, 'id');
+    response.status(201).json({ id });
```

Comment:
Can we talk about revising this from  `response.status(201).json({ id });` to `response.status(201).json({ id: id[0] });`?

For developer empathy I'd like to return it as `{ id: 1234}` instead of returning that nested in another object/array.  

If that works for you, we can revise it as we go (not worth delaying merging).  If you prefer it as is, I will need to revise my POST for consistency.